integration-tests/plugins/__tests__/inventory/order/order.js

```diff
@@ -160,6 +159,89 @@ describe("/store/carts", () => {
       )
     })
 
+    describe("canceling an order", () => {
+      it("cancels an order with many items should succeed", async () => {
```

Comment:
nit: Follow recent established test naming convention
```suggestion
      it("should cancel an order with many items", async () => {
```